tests/integration/remote/util.py

```diff
@@ -105,8 +107,15 @@ def do_ocean_tx_and_handle_gotchas(ocean, alice_wallet):
 
     print("Call create() from data NFT, and wait for it to complete...")
     try:
+        # Polygon & Mumbai uses EIP-1559. So, dynamically determine priority fee
+        gas_fees = requests.get("https://gasstation-mainnet.matic.network/v2").json()
```

Comment:
there is some duplication with the non_ocean_tx. Could we make a function that does the request and returns the priority fee and the max fee?

Could we also add some fallbacks, i.e. if the response from the API is not 200 or any key doesn't exist, we should handle that (maybe fail the tests gracefully?)

Reply:
Ok, I will also add a fallback for that response.